wee_slack.py

```diff
@@ -2400,6 +2401,11 @@ def resolve_ref(ref):
         # Something else, just return as-is
     return ref
 
+def create_user_status_string(profile):
+    status_emoji = profile.get("status_emoji") if profile.get("status_emoji") else "None"
+    status_text = profile.get("status_text") if profile.get("status_text") else "None"
+    return "[{}] {}".format(status_emoji, status_text)
```

Comment:
Won't this lead to `[None] None` as the topic if the other user has no emoji or text status? Wouldn't a blank topic be better?

Reply:
As another suggestion, I believe Slack's mention completion presents users as `*@username*  full_name | status_emoji status_text`. We could show `full_name | status_emoji status_text`.

Reply:
@rawdigits, @V13Axel : I think @auscompgeek's suggestion sounds good. Thoughts/opinions? It'd be good if we can get this closed out.